python/lsst/cp/pipe/cpLinearitySolve.py

```diff
@@ -1427,6 +1427,7 @@ def run(
         for amp in detector:
             ampName = amp.getName()
 
+            linearizer.inputGain[ampName] = inputPtc.gain[ampName]
```

Comment:
I think you need to add this also to the other linearity solve task (not just the new relative one).